requests/src/test/scala/uk/ac/wellcome/platform/api/requests/RequestingScenarioTest.scala

```diff
@@ -0,0 +1,80 @@
+package uk.ac.wellcome.platform.api.requests
+
+import akka.http.scaladsl.model.{HttpResponse, RequestEntity, StatusCodes}
+import org.scalatest.GivenWhenThen
+import org.scalatest.concurrent.IntegrationPatience
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.models.work.generators.ItemsGenerators
+import uk.ac.wellcome.platform.api.requests.fixtures.RequestsApiFixture
+import weco.api.stacks.services.memory.MemoryItemLookup
+import weco.catalogue.internal_model.identifiers.{IdState, IdentifierType}
+import weco.catalogue.internal_model.work.Item
+
+class RequestingScenarioTest
+    extends AnyFeatureSpec
+    with GivenWhenThen
+    with Matchers
+    with ItemsGenerators
+    with RequestsApiFixture
+    with IntegrationPatience {
+
+  Feature("requesting an item") {
+    Scenario("An item which is not from Sierra") {
+      Given("a physical item which is not from Sierra")
+      val item = createIdentifiedCalmItem
+
+      val lookup = new MemoryItemLookup(items = Seq(item))
+
+      withRequestsApi(lookup) { _ =>
```

Comment:
nit-shed: I think it's clearer to have the whole test within the fixture block so that Given/When/Then are maximally readable

Reply:
I agree.

Unfortunately you need to create the item before you can invoke the fixture block, so you can't do that yet. I'm hoping to wrangle the RequestsApi so I can bin the fixture block and have them all lined up (see also: the in-memory ItemLookup saving us an Elasticsearch fixture and the `waitForPostRequest` helper), but I couldn't quite get there in this PR.

I think that'll be easier when I actually start mucking around with SierraService, which is the only fixture-y thing this is still using.